plugins/tileacq.py

```diff
@@ -288,8 +295,6 @@ def show_dlg(self):
         else:
             logging.warning("Got unknown return code %s", ans)
 
-        # Don't hold references
```

Comment:
Why do you remove this? It shouldn't need to have a reference to the dialog after it's been destroyed.
Most likely, we need to unsubscribe from all the VAs of all the streams (ie, do the opposite of `_on_streams_change()`).